plugins/by-name/cord-nvim/default.nix

```
@@ -0,0 +1,26 @@
+{
+  lib,
+  ...
+}:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
```

```suggestion
}:
let
  inherit (lib.nixvim) defaultNullOpts;
  inherit (lib) types;
in
lib.nixvim.neovim-plugin.mkNeovimPlugin {
```

This will help removing a little bit of boilerplate.